applications/StructuralMechanicsApplication/custom_advanced_constitutive/generic_anisotropic_3d_law.cpp

```diff
@@ -86,7 +86,7 @@ void GenericAnisotropic3DLaw::CalculateMaterialResponsePK2(ConstitutiveLaw::Para
     Flags& r_flags = rValues.GetOptions();
 
     // Previous flags saved
-    const bool flag_strain       = r_flags.Is(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN);
+    //const bool flag_strain       = r_flags.Is(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN);
```

Comment:
So, at the end this only works for SD?

Reply:
not it is solved provisionally

Reply:
sorry but then it cannot be merged...

Reply:
Now it will work, besides this cl was motivated to be used in Small Strain. I've been thinking to be useful in LS